meson_options.txt

```diff
@@ -3,6 +3,8 @@ option('install-test', type: 'boolean', value: false)
 option('enable-tensorflow-lite', type: 'boolean', value: true)
 option('enable-tensorflow', type: 'boolean', value: true)
 option('enable-tensorflow-mem-optmz', type: 'boolean', value: true)
+option('enable-pytorch', type: 'boolean', value: true)
+option('enable-pytorch-use-gpu', type: 'boolean', value: false)
```

Comment:
We need some "explanation" for this option; readers may think GPU usage is deteremined at build-time. But, this is rather a default value that can be overriden at run-time. (adding a line of comment should suffice.)

Reply:
Done.